src/js/_enqueues/admin/site-icon.js

```diff
@@ -1,15 +1,40 @@
-(function($) {
-	var frame;
-
-	function calculateImageSelectOptions ( attachment ) {
-		var realWidth  = attachment.get( 'width' ),
+/**
+ * Handle the site icon setting in options-general.php.
+ *
+ * @since 6.5.0
+ * @output wp-admin/js/site-icon.js
+ */
+
+/* global jQuery, wp */
+
+( function( $ ) {
+	var $chooseButton = $( '#choose-from-library-button' ),
+	$iconPreview = $( '#site-icon-preview' ),
+	$browserIconPreview = $( '#browser-icon-preview' ),
+	$appIconPreview = $( '#app-icon-preview' ),
+	$hiddenDataField = $( '#site_icon_hidden_field' ),
+	$removeButton = $( '#js-remove-site-icon' ),
+	frame;
```

Comment:
```suggestion
	var $chooseButton = $( '#choose-from-library-button' ),
		$iconPreview = $( '#site-icon-preview' ),
		$browserIconPreview = $( '#browser-icon-preview' ),
		$appIconPreview = $( '#app-icon-preview' ),
		$hiddenDataField = $( '#site_icon_hidden_field' ),
		$removeButton = $( '#js-remove-site-icon' ),
		frame;
```

Variables after the first should be indented.

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#declaring-variables-with-var